src/Entities/Schema.php

```diff
@@ -102,6 +102,18 @@ public function __construct()
             'payment-cards/{cardId}' => function (array $content) {
                 return new PaymentCard($content);
             },
+            'paypal-accounts' => function (array $content) {
+                return new Collection(new PayPalAccount(), $content);
+            },
+            'paypal-accounts/{paypalAccountId}' => function (array $content) {
+                return new PayPalAccount($content);
+            },
+            'paypal-accounts/{paypalAccountId}/activation' => function (array $content) {
```

Comment:
I believe this builder and next (`paypal-accounts/{paypalAccountId}/deactivation`) does not make sense. And seems never used.